### ISSUE TITLE
the loss is becoming inf?

### ISSUE THREAD

**Issue by yangsuCV** [tensorlayer/openpose-plus#140]
the loss of the stage one (stage1/branch1, stage1/branch2) is becoming inf when the models_mobilenet is trained some few steps using parallel way in two gpu, however other stage's loss is normal  ,therefore the total loss  is inf  (python3.6,tensorflow 1.9.0 tensorlayer)。the train is normal when using cpu. I take same times to find bugs but failure. so I ask for help for ,thank you.

**Comment by yangsuCV** [comment]
Thank you for the prompt reply，god bless you.

**Comment by luomai** [comment]
Please let me know if this fixes your problem.

**Comment by yangsuCV** [comment]
the loss is normal after fixing the bug,thank you again.

**Comment by luomai** [comment]
Thanks for the feedback. Closing this issue then.
